src/services/api/UserService.ts

```diff
@@ -29,14 +30,21 @@ class UserService extends ApiService {
     formData.append('first_name', data.firstName);
     formData.append('last_name', data.lastName);
 
-    const { data: responseData } = await this.apiClient.post(ENDPOINTS.USER, formData);
+    const { data: responseData } = await this.apiClient.post(
+      ENDPOINTS.USER,
+      formData
+    );
 
     return responseData;
   };
 
   changePassword = async (data: object): Promise<void> => {
     this.apiClient.post(ENDPOINTS.CHANGE_PASSWORD, data);
   };
+
+  userPrefrencesUpdate = async (data: object): Promise<void> => {
+    this.apiClient.post(ENDPOINTS.USER_PREFRENCES, data);
+  };
```

Comment:
Already mentioned this, but we shouldn't use generic types like `object`... 
Please always create an interface...
By the way, what's this for?

```suggestion
  userPreferencesUpdate = async (data: object): Promise<void> => {
    this.apiClient.post(ENDPOINTS.USER_PREFERENCES, data);
  };
```

Reply:
Basically this is for the user to send the server his settings preferences, as in "I would like my account to prefer dark theme", i know that the dark theme switch is usually stored locally, however, in the future there might be user settings that would need to be stored on the server, if you think this is unnecessary i can replace it for local storage in 2 min :)

Reply:
yeah, let's use async storage... in case something like this is needed we can always add it.